clog/__init__.py

```diff
@@ -46,9 +46,18 @@
 from clog.loggers import ScribeLogger, ScribeIsNotForkSafeError
 from clog.global_state import log_line, reset_default_loggers
 
+uwsgi_plugin_enabled = False
+try:
+    from clog.uwsgi_plugin import uwsgi_patch_global_state, uwsgi_log_line
+    uwsgi_plugin_enabled = True
+except ImportError:
```

Comment:
Here we could do something like:
```python
if 'uwsgi' in sys.modules:
    ...
```
since uwsgi will inject itself before loading our app I believe (at least manually testing it appears to be so, and given the way the uwsgi python environment is initialized it makes sense). Logically it's a little more proactive than relying on the implicit failure of `import uwsgi` within the module, but I think it achieves the same goal. And python tends to follow the "easier to ask forgiveness than permission" philosophy no?